fsw/nos3_defs/tables/to_config.c

```diff
@@ -141,8 +142,8 @@ TO_ConfigTable_t to_ConfigTable =
        /* 40 - 49 */
        {GENERIC_CSS_HK_TLM_MID,       {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
        {GENERIC_CSS_DEVICE_TLM_MID,   {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
-       {TO_UNUSED_ENTRY,              {0,0},  0,   0x0000,     TO_GROUP_NONE,            0,0},
-       {TO_UNUSED_ENTRY,              {0,0},  0,   0x0000,     TO_GROUP_NONE,            0,0},
+       {GENERIC_RADIO_HK_TLM_MID,     {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
+       {GENERIC_RADIO_DEVICE_TLM_MID, {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
```

Comment:
Delete this line.  There is no GENERIC_RADIO_DEVICE_TLM_MID packet.

Reply:
Resolved in commit 38dd658